### ISSUE TITLE
How to locate some element, scroll to them one by one, and wait until they finish loading?

### ISSUE THREAD

**Issue by Edmounds** [scrapy-plugins/scrapy-playwright#166]
I'm new to scrapy.
I have used PageMethod,but it doesnt work well by using this
`PageMethod("evaluate", "window.scrollBy(0, 1000)"),
`
When it comes to this
`PageMethod("evaluate", "window.scrollBy(0, 1000)"),
PageMethod("wait_for_timeout", 2*1000),
PageMethod("evaluate", "window.scrollBy(0, 1000)"),
PageMethod("wait_for_timeout", 2*1000),
PageMethod("evaluate", "window.scrollBy(0, 1000)"),`

it didnt scroll like i think.

Then i try to use playwright_include_page,another quesion is:"Could it run Js?"

In conclusion,what should i do to realize " locate some element, scroll to them one by one, and wait until they finish loading"

**Comment by Gallaecio** [comment]
Are you trying to implement an “scroll to bottom” interaction? If so, see the scrapy-playwright example at https://docs.zyte.com/zyte-api/migration/browser-automation/index.html#consuming-scroll-based-pagination.

**Comment by Edmounds** [comment]
> Are you trying to implement an “scroll to bottom” interaction? If so, see the scrapy-playwright example at https://docs.zyte.com/zyte-api/migration/browser-automation/index.html#consuming-scroll-based-pagination.

Many thanks
